source/s3_meta_request.c

```diff
@@ -1624,12 +1629,14 @@ void aws_s3_meta_request_send_request_finish_default(
 
     AWS_LOGF_DEBUG(
         AWS_LS_S3_META_REQUEST,
-        "id=%p: Request %p finished with error code %d (%s) and response status %d",
+        "id=%p: Request %p finished with error code %d (%s) and response status %d, request id: %s, amz_id_2: %s",
```

Comment:
trivial: Refer to both of these by either: their full literal header name, or their modeled names ... actually apparently neither of these are in the service model

```suggestion
        "id=%p: Request %p finished with error code %d (%s) and response status %d, x-amz-request-id: %s, x-amz-id-2: %s",
```